netty/src/test/java/io/grpc/netty/UdsNettyChannelProviderTest.java

```diff
@@ -36,22 +36,25 @@
 import io.grpc.testing.protobuf.SimpleResponse;
 import io.grpc.testing.protobuf.SimpleServiceGrpc;
 import io.netty.channel.EventLoopGroup;
-import io.netty.channel.epoll.Epoll;
 import io.netty.channel.epoll.EpollEventLoopGroup;
 import io.netty.channel.epoll.EpollServerDomainSocketChannel;
 import io.netty.channel.unix.DomainSocketAddress;
 import java.io.IOException;
+import org.junit.After;
 import org.junit.Assume;
 import org.junit.Rule;
 import org.junit.Test;
+import org.junit.rules.TemporaryFolder;
 import org.junit.runner.RunWith;
 import org.junit.runners.JUnit4;
 
 /** Unit tests for {@link UdsNettyChannelProvider}. */
 @RunWith(JUnit4.class)
 public class UdsNettyChannelProviderTest {
 
-  private static final String TEST_SOCKET = "/tmp/test.socket";
+  @Rule public TemporaryFolder tempFolder = new TemporaryFolder();
+
+  private static final String TEST_SOCKET = "/test.socket";
```

Comment:
Remove? It seems it is only used one place and can't really be understood without looking at where it is used (because of the leading slash).